llmproxy/llmproxy.py

```diff
@@ -181,12 +182,15 @@ def route(
     def _cost_route(self, prompt: str):
         min_heap = MinHeap()
         for model_name, instance in self.user_models.items():
-            logger.info(msg="========Start Cost Estimation===========")
-            cost = instance.get_estimated_max_cost(prompt=prompt)
-            logger.info(msg="========End Cost Estimation===========\n")
+            try:
+                logger.info(msg="========Start Cost Estimation===========")
+                cost = instance.get_estimated_max_cost(prompt=prompt)
+                logger.info(msg="========End Cost Estimation===========\n")
 
-            item = {"name": model_name, "cost": cost, "instance": instance}
-            min_heap.push(cost, item)
+                item = {"name": model_name, "cost": cost, "instance": instance}
```

Comment:
<img width="851" alt="Screenshot 2024-01-20 at 11 42 27 PM" src="https://github.com/llm-proxy/llm-proxy/assets/46403906/0ceb2392-ac91-4a5d-b414-ff4c1a21bd91">
does this deal with this? when running test.py